src/Protocol/ActivityPub/Processor.php

```diff
@@ -602,6 +602,12 @@ public static function postItem(array $activity, array $item)
 				continue;
 			}
 
+			if (!$item['isForum'] && ($receiver != 0) && ($item['gravity'] == GRAVITY_PARENT) &&
+				($item['post-reason'] == Item::PR_BCC) && !Contact::isSharingByURL($activity['author'], $receiver)) {
+				Logger::info('Top level post via BCC from a non follower, ignoring', ['uid' => $receiver, 'contact' => $item['contact-id']]);
```

Comment:
Do you mean from an account that isn't followed?

Reply:
I'm always confused with follower, following, sharing and so on :-) So: How should I name it?

Reply:
in my mind, it is always a Stalker... no you should not call it that way.

The follow you, but you don't follow them. I'd say

> Top level post via BCC from an account that isn't followed, ignoring

Reply:
I'm with Tobias.